src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private int index = new Random().nextInt(Colors.values().length);
+
     public String getRandomColor() {
-        return null;
+        return String.valueOf(Colors.values()[index]);
```

Comment:
Давай генеррировать индекс динамически, 
```suggestion
        return String.valueOf(Colors.values()[random.nextInt(Colors.values().length)]);
```